docs/rules_list.rst

```diff
@@ -7,7 +7,26 @@ Rules list
 This is the complete list of all Robocop rules grouped by categories.
 If you want to learn more about the rules and their features, see :ref:`rules`.
 
+There are over a 100 rules available in Robocop and they are organized into the following groups:
+
+* 01: Base
+* 02: :ref:`Documentation`
+* 03: :ref:`Naming`
+* 04: :ref:`Errors`
+* 05: :ref:`Lengths`
+* 06: :ref:`Tags`
+* 07: :ref:`Comments`
+* 08: :ref:`Duplications`
+* 09: :ref:`Misc`
+* 10: :ref:`Spacing`
+
+Each category has a corresponding 2-digit ID that is a prefix for the 4-digit rule ID.
```

Comment:
It sounds like ID is 6 digits while it is 4. Maybe we should reword this sentence, ie "Each rule has 4-digit ID which starts with corresponding 2-digit category ID"

Reply:
Updated.
BTW I was thinking how we should name these parts and maybe something like this?
- rule ID - a 4-digit ID that contains:
  - 2-digit category ID
  - 2-digit rule number

"rule ID" - because we `rule_id` in code
"category ID" - because many checkers may relate to the same category, so checker ID is not a precise name, that's why we should use "category" here
"rule number" - because "rule ID" is already taken, but this is actually just a consecutive number of the rule

WDYT?

Reply:
Good idea, using bullet list make it more clear and category is better than checker id (like you wrote, we have multiple checkers in single category already). And we use ordered number rarher than any unique id so its also better name :)

Reply:
Updated.